src/test/java/seedu/guest/logic/commands/ClearCommandTest.java

```diff
@@ -13,15 +13,15 @@
 public class ClearCommandTest {
 
     @Test
-    public void execute_emptyAddressBook_success() {
+    public void execute_emptyGuestBook_success() {
         Model model = new ModelManager();
         Model expectedModel = new ModelManager();
 
         assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
     @Test
-    public void execute_nonEmptyAddressBook_success() {
+    public void execute_nonEmptyGuestBook_success() {
         Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
Should this be getTypicalGuestBook() as well? Found in seedu.address.testutil.TypicalPersons

Reply:
This should be refactored under Zong Xian's (your) part as it is this under the testutil package.